sdks/python/apache_beam/transforms/display.py

```diff
@@ -137,9 +137,12 @@ def create_payload(dd):
       except ValueError:
         # Skip if the display data is invalid.
         return None
-      if 'value' not in display_data_dict or 'label' not in display_data_dict:
+      if 'value' not in display_data_dict or (
```

Comment:
1. should this be `if 'value' not in display_data_dict` ***and***  ... ?

I think you could rewrite it as:
```
try:
  label = (
          display_data_dict['label']
          if 'label' in display_data_dict else display_data_dict['key'])
  value = display_data_dict['value']
except KeyError:
  return None
```

2. Where is the expected structure of display_data_dict documented? Why these particular keys in DD dictionary have specific treatment? Should we assert that `label` and `key` don't appear at the same time?
3. should we update display_test as well?

Reply:
Some existing  transforms, sources etc. do not set a 'label'. So we'll have to default to 'key' (which is always set)  to not loose DisplayData for some existing transforms.

Added a test and comments and refactored the code.